core/vtk/ttkTriangulationRequest/ttkTriangulationRequest.h

```diff
@@ -87,58 +65,32 @@ class TTKTRIANGULATIONREQUEST_EXPORT ttkTriangulationRequest
   vtkSetMacro(KeepAllDataArrays, bool);
   vtkGetMacro(KeepAllDataArrays, bool);
 
-  vtkSetMacro(PeriodicBoundaryConditions, int);
-  vtkGetMacro(PeriodicBoundaryConditions, int);
-
-  int FillInputPortInformation(int port, vtkInformation *info) override {
-
-    switch(port) {
-      case 0:
-        info->Set(vtkDataObject::DATA_TYPE_NAME(), "vtkDataSet");
-        break;
-
-      default:
-        break;
-    }
-
-    return 1;
-  }
-
-  int FillOutputPortInformation(int port, vtkInformation *info) override {
-
-    switch(port) {
-      case 0:
-        info->Set(vtkDataObject::DATA_TYPE_NAME(), "vtkUnstructuredGrid");
-        break;
-
-      default:
-        break;
-    }
-
-    return 1;
-  }
-
 protected:
   ttkTriangulationRequest() {
-    UseAllCores = true;
     SimplexType = 0;
     SimplexIdentifier = 0;
     RequestType = 0;
     KeepAllDataArrays = true;
-    PeriodicBoundaryConditions = false;
 
     SetNumberOfInputPorts(1);
     SetNumberOfOutputPorts(1);
+
+    this->setDebugMsgPrefix("ttkTriangulationRequest");
```

Comment:
to follow the convention used in the rest of the code, you can simply go for ```TriangulationRequest``` (instead of ```ttkTriangulationRequest```).